### DISCUSSION TITLE
Error setting up a cron job: "cron: can't lock /var/run/crond.pid... Resource temporarily unavailable"

### DISCUSSION THREAD

**Discussion by gittbuh** [firefly-iii/firefly-iii#6223] [Firefly III]
Hi all, 

I am trying to set up a cron job in order for recurring transactions to work, following the documentation. Whether I try calling a command or requesting a page over the web I get the following error: "cron: can't lock /var/run/crond.pid, otherpid may be 100: Resource temporarily unavailable".

I would appreciate some help :)

Thanks a lot.

**Comment by JC5** [comment]
I've never seen this one before. But how exactly are you trying to call a command? As part of the cron job?

**Comment by gittbuh** [comment]
Hi, thanks for your reply. 

I basically copy-pasted the two lines showed in the documentation directly into the command line:

cron job for Firefly III
0 3 * * * /usr/bin/php /var/www/html/artisan firefly-iii:cron

This is inside a lxc container in Proxmox. 

I guess this is probably not correct?

If a type cron (+ enter) in the command line, I get the same error.

**Reply by gittbuh** [reply]
Hi again,

Sorry,  I think I went through the documentation a bit too fast. I had never done this before, messing with cron.

So I typed crontab -e in the command line, opened nano editor and copy-pasted those two lines inside at the very bottom, saved and out. Will this work now? How to test? 

Thanks!

**Reply by JC5** [reply]
That should work. Just wait until tomorrow to see the results in your system's logs.
